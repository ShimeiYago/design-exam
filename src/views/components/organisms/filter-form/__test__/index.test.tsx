import { shallow, ShallowWrapper } from 'enzyme';
import { FilterForm, Props } from '..';

let wrapper: ShallowWrapper<Props>;

const props: Props = {
  onChange: jest.fn(),
};

describe('Shallow Snapshot Tests', () => {
  beforeEach(() => {
    wrapper = shallow(<FilterForm {...props} />);
  });

  it('basic', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
