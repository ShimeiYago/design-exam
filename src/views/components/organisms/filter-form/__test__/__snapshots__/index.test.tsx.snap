// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Shallow Snapshot Tests basic 1`] = `
<Spacing
  paddingHorizontal="32px"
  paddingVertical="24px"
>
  <TextWrapper
    bold={true}
    color="gray"
    size="medium"
  >
    フィルター
  </TextWrapper>
  <Spacing
    key="filtering-表示分類"
    marginTop="32px"
  >
    <RadioGroup
      groupLabel="表示分類"
      onChange={[Function]}
      options={
        Array [
          Object {
            "label": "進学",
            "value": "1",
          },
          Object {
            "label": "就活",
            "value": "2",
          },
        ]
      }
      selectedValue="1"
    />
  </Spacing>
  <Spacing
    key="filtering-進学・就職先"
    marginTop="32px"
  >
    <RadioGroup
      groupLabel="進学・就職先"
      onChange={[Function]}
      options={
        Array [
          Object {
            "label": "地元進学",
            "value": "0",
          },
          Object {
            "label": "流出",
            "value": "1",
          },
          Object {
            "label": "流入",
            "value": "2",
          },
          Object {
            "label": "純流入",
            "value": "3",
          },
        ]
      }
      selectedValue="0"
    />
  </Spacing>
  <Spacing
    key="filtering-表示内容"
    marginTop="32px"
  >
    <RadioGroup
      groupLabel="表示内容"
      onChange={[Function]}
      options={
        Array [
          Object {
            "label": "すべての進学",
            "value": "10",
          },
          Object {
            "label": "大学進学",
            "value": "11",
          },
          Object {
            "label": "短期大学進学",
            "value": "12",
          },
        ]
      }
      selectedValue="10"
    />
  </Spacing>
  <Spacing
    key="filtering-性別"
    marginTop="32px"
  >
    <RadioGroup
      groupLabel="性別"
      onChange={[Function]}
      options={
        Array [
          Object {
            "label": "総数",
            "value": "0",
          },
          Object {
            "label": "男性",
            "value": "1",
          },
          Object {
            "label": "女性",
            "value": "2",
          },
        ]
      }
      selectedValue="0"
    />
  </Spacing>
</Spacing>
`;
